spelling.json

```diff
@@ -0,0 +1 @@
data/heman/test/stb_image_resize.h"}]}, "upsamples": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "videogame": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "premultiplying": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "unpremultiplying": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "premultiplication": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "reblocking": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_resize.h"}]}, "jidctint": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "jpegs": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "preshift": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "cinfo": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "exitting": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "hres": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "vres": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "colorsused": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "mspaint": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "precessing": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "pngchunk": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "preceeded": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "dword": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "gifs": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "sscanf": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image.h"}]}, "memmove": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "realloc": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "simplicitly": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "sbcount": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "sbpush": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "stbiw": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "adler": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}, "freeable": {"files": [{"category": "text", "file": "/home/tgates/data/heman/test/stb_image_write.h"}]}}
```

Comment:
This file has paths like `/home/tgates` so probably should not be checked in.

Reply:
oh sorry! thx removed